test/integration/apiserver/patch_test.go

```diff
@@ -43,14 +41,30 @@ func TestPatchConflicts(t *testing.T) {
 	ns := framework.CreateTestingNamespace("status-code", s, t)
 	defer framework.DeleteTestingNamespace(ns, s, t)
 
-	// Create the object we're going to conflict on
-	clientSet.CoreV1().Secrets(ns.Name).Create(&v1.Secret{
+	numOfConcurrentPatches := 2 * handlers.MaxRetryWhenPatchConflicts
+
+	UIDs := make([]types.UID, numOfConcurrentPatches)
+	ownerRefs := []metav1.OwnerReference{}
+	for i := 0; i < numOfConcurrentPatches; i++ {
+		uid := types.UID(uuid.NewRandom().String())
```

Comment:
are we running GC in this setup? will missing ownerRefs trigger deletion of the secret?